src/Modules/OrchardCore.Commerce/Views/ShoppingCart/Index.cshtml

```diff
@@ -10,9 +10,15 @@
 
 <section id="shopping-cart">
     <shape type="ShoppingCartTable" prop-ShoppingCartViewModel="@Model"></shape>
-    @if (Model.IsInvalid)
+    @if (Model.InvalidReasons.Any())
     {
-        <p>@T["Checkout unavailable — an item is out of stock."]</p>
+        <div class="checkout-unavailable">@T["Checkout unavailable due to the following:"]</div>
```

Comment:
```suggestion
        <div class="checkout-unavailable">@T["Checkout unavailable due to the following reasons:"]</div>
```
I think you need that extra word here? At least I remember seeing that specific phrasing in the past.